src/components/tariffs-trend-card.tsx

```diff
@@ -252,12 +246,12 @@ export const TariffsTrendCardMinified: React.FC<
         <Typography variant="h3">{cardTitle}</Typography>
         <Typography variant="body2">{cardDescription}</Typography>
         <TariffsTrendChart
-          id={operatorId}
-          operatorLabel={operatorLabel}
-          observations={chartData}
-          netTariffs={restNetTariffs}
-          viewBy={filters?.viewBy ?? "progress"}
-          compareWith={filters?.compareWith ?? []}
+          id={logic.operatorId}
+          operatorLabel={logic.operatorLabel}
+          observations={logic.chartData}
+          netTariffs={logic.restNetTariffs}
```

Comment:
Seems like logic should be named chartData and logic.chartData called chartData.observations ?